src/test/c_api_stress_tests.cpp

```diff
@@ -278,7 +278,7 @@ class StressPipelineCustomNodesWithPreallocatedBuffersCapiConfigChanges : public
     void checkInferResponse(OVMS_InferenceResponse* response) override {
         ASSERT_NE(response, nullptr);
         uint32_t outputCount = 42;
-        ASSERT_CAPI_STATUS_NULL(OVMS_InferenceResponseGetOutputCount(response, &outputCount));
+        ASSERT_CAPI_STATUS_NULL(OVMS_InferenceResponseOutputCount(response, &outputCount));
```

Comment:
This is probably already fixed on main - PR not rebased?

Reply:
Fix was cherry-picked on this branch as well